graylog2-inputs/src/main/java/org/graylog2/inputs/transports/UdpTransport.java

```diff
@@ -77,11 +62,11 @@ private static Executor executorService(final String executorName, final String
 
     @Override
     public Bootstrap getBootstrap() {
-        final ConnectionlessBootstrap bootstrap =
-                new ConnectionlessBootstrap(new NioDatagramChannelFactory(workerExecutor));
+        final ConnectionlessBootstrap bootstrap = new ConnectionlessBootstrap(new NioDatagramChannelFactory(workerExecutor));
 
-        bootstrap.setOption("receiveBufferSizePredictorFactory", new FixedReceiveBufferSizePredictorFactory(8192));
-        bootstrap.setOption("receiveBufferSize", getRecvBufferSize());
+        final int recvBufferSize = Ints.saturatedCast(getRecvBufferSize());
```

Comment:
I think that would be good. That would also deal with the problem that our previous receive buffer default for UDP was 1MB.